seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-assert/src/main/java/org/apache/seatunnel/flink/assertion/sink/AssertSink.java

```diff
@@ -71,7 +73,7 @@ public void outputStream(FlinkEnvironment env, DataStream<Row> dataStream) {
                     throw new IllegalStateException("row :" + row + "field name of the fail rule: " + failRule.getFieldName());
                 });
             return null;
-        });
+        }).print();
```

Comment:
You don't need to add print() in stream mode.
```suggestion
        });
```